boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/liberty/LibertyPackageMojo.java

```diff
@@ -292,10 +298,21 @@ private void createServer() throws MojoExecutionException {
      * Invoke the liberty-maven-plugin to run the install-app goal.
      */
     private void installApp(String installAppPackagesVal) throws MojoExecutionException {
-        executeMojo(getPlugin(), goal("install-apps"),
-                configuration(element(name("installAppPackages"), installAppPackagesVal),
-                        element(name("serverName"), libertyServerName), getRuntimeArtifactElement()),
-                getExecutionEnvironment());
+
+		if (MavenProjectUtil.isNotNullOrEmpty(springBootVersion)) { 
```

Comment:
Also, can more of the code in this method be reused? Much of the config in both cases is the same.

Reply:
> 
> 
> Fix spacing. Also instead of checking `springBootVersion` you can just check the `installAppPackagesVal`.

If you check the value probably worth making a constant.... (which eventually could even be factored into ci.maven/ci.common maybe since that's where it really comes from..but I'm not suggesting we do a ci.* change now).

Reply:
Agree, already gave the suggestion to Andy over Slack.

Reply:
not seeing right now how to re-use this code block - seems like they are different invocations of the executeMojo with different params dependent on what type of app we are packaging. But I am probably just not thinking about it correctly. I will continue to look at this...